lib/screens/questionaire_screen/widgets/cover.dart

```diff
@@ -21,76 +21,99 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 import '../../../themes/global_theme.dart';
 import '../../../widgets/picos_add_button_bar.dart';
-import '../../../widgets/picos_body.dart';
 import '../../../widgets/picos_ink_well_button.dart';
 
 /// A cover for the different sections of the questionaire.
 class Cover extends StatelessWidget {
   /// Constructs the cover page.
   const Cover({
     this.title = '',
+    this.image = '',
     Key? key,
     this.backFunction,
     this.nextFunction,
   }) : super(key: key);
 
   /// The title cover.
   final String title;
+
+  /// Contains the annotation for the image.
+  final String image;
+
   /// Function for getting a page back.
   final void Function()? backFunction;
+
   /// Function for getting the next page.
   final void Function()? nextFunction;
 
   @override
   Widget build(BuildContext context) {
     final String back = AppLocalizations.of(context)!.back;
     final String next = AppLocalizations.of(context)!.next;
+    final String start = AppLocalizations.of(context)!.letsStart;
     final GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
     final double height = MediaQuery.of(context).size.height;
 
-    return PicosBody(
+    return Container(
```

Comment:
This reintroduces the overflow problem I've fixed with PicosBody.
You'll notice the problem if you do the following:

1. Comment out line 197 and 211 in questionaire_screen.dart.
2. Open the application and go into the questionaire.
3. Select some input where the keyboard shows up.
4. Move to a cover page with the keyboard open.

- Comment out line 197 and 211 in questionaire_screen.dart